test/shopify_app/controller_concerns/login_protection_test.rb

```diff
@@ -116,42 +112,6 @@ class LoginProtectionControllerTest < ActionController::TestCase
     end
   end
 
-  test "#current_shopify_session loads offline session if token is signed with new secret" do
```

Comment:
@tatsuya I noticed you added this test - can you clarify what you thought this test was trying to cover since `secret` and `old_secret` in the configuration object does not change the behavior of whether an online or offline token is used

Reply:
@teddyhwang The intention here was to ensure that `LoginProtection#current_shopify_session` works properly with session tokens signed with real `secret` and `old_secret`. It didn't mean to test the differences between online or offline tokens.

In the past, this method was returning a `nil` when a token was signed with an old secret. After [the fix](https://github.com/Shopify/shopify-api-ruby/issues/978) was made in the shopify_api lib, that was no longer the case though, I thought having a test case like this will help to ensure that future upgrades to shopify_app lib do not break the current behaviour. Full context https://github.com/Shopify/shopify_app/pull/1479

Reply:
Having said that, no strong opinion for it. Feel free to keep it or remove it 👍